velocity_multiplier:          1.5

controller_mode:              "joy"

joy_sub_topic:                "joy"
cmd_vel_sub_topic:            "cmd_vel"

mavros_local_pos_sub_topic:   "mavros/local_position/pose"
mavros_state_sub_topic:       "mavros/state"
mavros_position_sub_topic:    "mavros/vision_pose/pose"

mavros_waypoint_pub_topic:    "mavros/setpoint_position/local"
